### ISSUE TITLE
Searchs with no results shouldn't be a error

### ISSUE THREAD

**Issue by rul** [smira/aptly#334]
Currently whenever you search for something that it's not in the repository, aptly will finish with exit code 1.

```
$ aptly repo search myrepo 'Name (carlitos)'
ERROR: no results
$ echo $?
1
```

Not having packages matching a query is not an error, so aptly should finish with a nice and graceful exit code 0.

**Comment by smira** [comment]
Well, non-zero exit status is quite common for `search`-like commands:

```
EXIT STATUS
     The grep utility exits with one of the following values:

     0     One or more lines were selected.
     1     No lines were selected.
```

**Comment by rul** [comment]
Yes, you're right! I'm more worried about the output to `stdout` than about the exit code, though.

**Comment by smira** [comment]
It was fixed in #249
